Taskfile.yml

```diff
@@ -94,7 +94,7 @@ tasks:
       - "clp-py-utils"
       - "init"
       - "job-orchestration"
-      - "log-viewer-webui"
+      - "log-viewer-webui-clients"
```

Comment:
This task should also depend on the log-viewer-webui-clients checksum as well as the server source files, right?

Reply:
We also want to add the server source files as dependencies, right?